### PR TITLE
Changed ControlNet read and write to use the new ControlNetVersioner

### PR DESCRIPTION
**Pull request by jessemapel**
Also did some clean up in ControlNetVersioner

### CODE REVIEW

**Comment by jessemapel** [comment]
[Change ControlNet readControl and write methods](https://trello.com/c/GmkUgmiZ/148-change-controlnet-readcontrol-and-write-methods)

**Comment by jessemapel** [comment]
[Change ControlNet readControl and write methods](https://trello.com/c/GmkUgmiZ/148-change-controlnet-readcontrol-and-write-methods)

**Comment by jessemapel** [comment]
Don't merge yet
fixing build errors

**Comment by jessemapel** [comment]
Now it can be reviewed and merged.  We're still not completely building because of ControlNetDiff.  See this trello card (https://trello.com/c/nARtLxP3)

File: isis/src/control/objs/ControlNet/ControlNet.cpp [hunk 1]
```diff
@@ -288,41 +265,41 @@ namespace Isis {
...
+      }
+      catch (IException &e) {
+        QString msg = "Failed to convert control network to Pvl format.";
+        throw IException(e, IException::Programmer, msg, _FILEINFO_);
+      }
 
-      ControlPointFileEntryV0002 pointFileEntry = point->ToFileEntry();
-      fileDataPoints.append(pointFileEntry);
+      try {
+        network.write(ptfile);
```
Comment (kberryUSGS) [inline isis/src/control/objs/ControlNet/ControlNet.cpp]: I think this needs to be `FileName(ptfile)` since cnv::write takes a `FileName`

Comment (jessemapel) [reply isis/src/control/objs/ControlNet/ControlNet.cpp]: This is Pvl::write which oddly enough takes a qstring for the file name

Comment (kberryUSGS) [reply isis/src/control/objs/ControlNet/ControlNet.cpp]: Oh whoops, nevermind then!

**Pull request merged**